packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/ManagedACTAccessRequirementItemView.tsx

```diff
@@ -0,0 +1,305 @@
+            severity={'success'}
+          >
+            <strong>Your data access request has been approved.</strong>
+          </Alert>
+        )}
+      {accessRequirementStatus?.currentSubmissionStatus?.state ===
+        SubmissionState.APPROVED &&
+        !isApproved && (
+          <Alert
+            className={'access-requirement-list-alert'}
+            severity={'warning'}
+          >
+            <Typography variant={'smallText1'} gutterBottom>
+              <strong>Your data access has been revoked.</strong>
+            </Typography>
+            {accessRequirementStatus.expiredOn &&
+              dayjs(accessRequirementStatus.expiredOn).isBefore(dayjs()) && (
+                <Typography variant={'smallText1'} gutterBottom>
+                  Your access expired on{' '}
+                  {formatDate(dayjs(accessRequirementStatus.expiredOn))}.
+                </Typography>
+              )}
+          </Alert>
+        )}
+      {accessRequirementStatus?.currentSubmissionStatus?.state ===
+        SubmissionState.REJECTED && (
+        <Alert className={'access-requirement-list-alert'} severity={'error'}>
+          <Typography variant={'body1'} gutterBottom>
+            <strong>Your data access request has been rejected.</strong>
+          </Typography>
+          <Typography
+            variant={'body1'}
+            gutterBottom
+            sx={{ whiteSpace: 'pre-line' }}
+          >
+            {accessRequirementStatus?.currentSubmissionStatus.rejectedReason ||
+              ''}
+          </Typography>
+        </Alert>
+      )}
+      {accessRequirementStatus?.currentSubmissionStatus?.state ===
+        SubmissionState.CANCELLED && (
+        <Alert className={'access-requirement-list-alert'} severity={'info'}>
+          <Typography variant={'body1'}>
+            Your data access request has been canceled.
+          </Typography>
+        </Alert>
+      )}
+    </Stack>
+  )
+
+  let actions: ButtonProps[] = []
+  if (!isLoading && accessRequirementStatus) {
+    actions = getActionButtonProps(
+      accessRequirementStatus,
+      isExempt,
+      cancelRequestIsPending,
+      {
+        onRequestAccess,
+        onRejectTerms,
+        onCancelDataAccessRequest,
+      },
+    )
+  }
+
+  let requirementItemState: RequirementItemStatus =
+    RequirementItemStatus.LOADING
+  if (!isLoading && accessRequirementStatus) {
+    requirementItemState = getRequirementItemUiState(
+      accessRequirementStatus,
+      isExempt,
+    )
+  }
+
+  return (
+    <RequirementItem
+      data-testid="ManagedACTAccessRequirementItem"
+      status={requirementItemState}
+      actions={actions}
+    >
+      {/*  If not approved/exempt, show the terms */}
+      {!isApproved && !isExempt && <>{requirementTermsWikiContent}</>}
+      {(isApproved || isExempt) && (
+        <>
+          <Typography variant={'body1'}>
+            {isApproved && APPROVED_TEXT} {isExempt && EXEMPT_TEXT}{' '}
```

Comment:
Now handle exempt case:

![image](https://github.com/user-attachments/assets/6afe163a-44ba-4084-af34-8cb94549745b)


It is technically possible to be both approved and exempt, which is fine:

![image](https://github.com/user-attachments/assets/dec5d054-a480-435f-a72f-e8757f662beb)